recipes/_configure_server.rb

```diff
@@ -33,6 +36,13 @@
   commit_retry_time
 ).freeze
 
+ruby_block 'set pass4SymmKey and sslKeysfilePassword' do
+  block do
+    server_stanzas['general']['pass4SymmKey'] = CernerSplunk.splunk_encrypt_password default_pass4symmkey, node.run_state['cerner_splunk']['splunk.secret']
```

Comment:
This block won't execute until converge time, so these need to use `||=` instead of `=`